src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.container.ts

```diff
@@ -4,14 +4,24 @@ import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors
 import { RootState } from 'modules/common/types'
 import { saveItemRequest, SAVE_ITEM_REQUEST } from 'modules/item/actions'
 import { getLoading, getError } from 'modules/item/selectors'
-import { MapStateProps, MapDispatchProps, MapDispatch } from './CreateSingleItemModal.types'
+import { getCollection } from 'modules/collection/selectors'
+import { Collection, CollectionType } from 'modules/collection/types'
+import { MapStateProps, MapDispatchProps, MapDispatch, OwnProps } from './CreateSingleItemModal.types'
 import CreateSingleItemModal from './CreateSingleItemModal'
+import { getCollectionType } from 'modules/collection/utils'
 
-const mapState = (state: RootState): MapStateProps => ({
-  address: getAddress(state),
-  error: getError(state),
-  isLoading: isLoadingType(getLoading(state), SAVE_ITEM_REQUEST)
-})
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  const collection: Collection | null = ownProps.metadata.collectionId ? getCollection(state, ownProps.metadata.collectionId) : null
+  const collectionUrn: string | null =
+    collection?.urn && getCollectionType(collection) === CollectionType.THIRD_PARTY ? collection.urn : null
```

Comment:
Same comment for this variable as for [CreateMultipleItemsModal.container.ts](https://github.com/decentraland/builder/pull/1776/files#diff-75031743d9ed991a0b5782130bf9e81ef4a2403a3a6f2271164ae6a6390fb80a). Plus if you check [here](https://github.com/decentraland/builder/pull/1776/files#diff-14a508dde6f9bf5af42a88ac3c6859b51313837d30291f7c8cee96199399e024R189) it's already confusing as the `isThirdParty` check should be unnecessary

Reply:
Changed it to be the same as as the `CreateMultipleItemModal` one.